src/Http/Requests/PageCreateRequest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace EscolaLms\Pages\Http\Requests;
+
+use EscolaLms\Core\Models\User;
+use Illuminate\Foundation\Http\FormRequest;
+
+class PageCreateRequest extends FormRequest
+{
+    protected function prepareForValidation()
+    {
+        $this->merge(['slug' => $this->route('slug')]);
+    }
+
+    /**
+     * @return bool
+     */
+    public function authorize()
+    {
+        /** @var User $user */
+        $user = $this->user();
+        return $user!=null && $user->can('create:pages', 'api');
```

Comment:
`!==`, try to use as little `!=` and `==` as you can

Reply:
Replaced with a strict comparison operator.